api/api_sources/sources/database/database-schema/schema-sqls/SpeciesAgencyCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: species_agency_code ### --
+
+        
+CREATE TABLE species_agency_code ();
+ALTER TABLE species_agency_code ADD COLUMN species_agency_code_id SERIAL PRIMARY KEY;
+ALTER TABLE species_agency_code ADD COLUMN species_agency_code VARCHAR(10) NOT NULL UNIQUE;
+ALTER TABLE species_agency_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE species_agency_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE species_agency_code IS 'Code table for schema of species agencies';
```

Comment:
I'd like to recommend we add a more descriptive table comment that describes an "agency" from a business perspective.

For example; The Agency that oversees a particular activity concerning the **Invasive Alien Plant Program**.  It is the originating agency that data belongs to or the agency that funded the activity.

Note: We may want to use something more generic than "Invasive Alien Plant Program". Let's ask Mike.